assets/templates/dining_lists/dining_slot_info.html

```diff
@@ -70,15 +85,17 @@ <h3>{{ dining_list.dish }}</h3>
     </div>
 
     {# Change buttons #}
-    {% if can_change_settings %}
+    {% if dining_list|is_owner:user %}
         <div class="row mt-3">
             <div class="col-md">
                 {% url 'slot_change' day=date.day month=date.month year=date.year identifier=dining_list.association.slug as url %}
                 <a class="btn btn-primary btn-block" href="{{ url }}">
                     <i class="fas fa-tools"></i> Change information
                 </a>
             </div>
-            {% if can_remove_list %}
+            {% if dining_list|is_owner:user and dining_list.diners.count < dining_list.min_diners %}
+                {# The min_diners check is only performed here and not while a list is actually deleted, so it's #}
+                {# possible to delete a list even if there are more diners than min_diners XD #}
```

Comment:
Haha, nice catch.
Speaking of which, I have been toying with the idea of setting it fixed to 6 instead of user-determined as it rarely is an issue and thus specefied by the owner. What do you think?

Reply:
I have earlier written about this here: #127. I suggest you read that and comment what you think, I'm curious to hear your opinion about that :smiley:. The gist is that I had the same idea kinda.